UC-Berkeley-Extension/global
# Layout builder

## Description

Sets up Layoput Builder for testing on Articles.

## Testing instructions
1. Checkout this branch
1. Run `composer install`
2. Build the site
    * From inside the VM, run `/var/www/html/scripts/setup.sh`
3. Login as admin 
4. Create an article -- set to published
5. Change the layout for that article using the Layout tab.
6. View the changes
7. Change the layout using the layout tab -- set the state to Draft -- save the layout change
8. View the node and notice that the revision changes have not been applied
9. Publish the layout changes
6. View the changes on the published node

- Then run through these steps using the steps from #28 and #29 to test as a non-admin user.

## Remaining tasks
_Delete this section if there are no remaining tasks._
